src/flutter/shell/platform/linux_embedded/window/elinux_window_wayland.cc

```diff
@@ -1199,7 +1218,17 @@ bool ELinuxWindowWayland::CreateRenderSurface(int32_t width_px,
       std::make_unique<EnvironmentEgl>(wl_display_)));
   render_surface_->SetNativeWindow(native_window_.get());
 
-  if (view_properties_.use_window_decoration) {
+  // Setup Server Side Decorations if supported, using xdg-decoration
+  // Otherwise Fallback to Drawn Windows if flag is set
+  if (decoration_manager_ && xdg_toplevel_) {
```

Comment:
The main target of flutter for elinux is embedded devices. We don't usually use window decoration. So, this option should be disabled by default. We had that option for use cases that we would use when developing on PC, but not on the device.